src/clr-angular/utils/_mixins.clarity.scss

```diff
@@ -95,6 +95,27 @@ $clr-outline-spread: 2px;
   }
 }
 
+@mixin equilateral($size: 1.2rem) {
+  height: $size;
+  width: $size;
+}
+
+@mixin forcemin-equilateral($size: 1.2rem) {
```

Comment:
nit: name both `min-equilateral` and `max-equilateral` instead of using force.

Reply:
Sounds good.